xdsl/dialects/riscv_snitch.py

```diff
@@ -462,6 +463,8 @@ class DMSourceOp(IRDLOperation, RISCVInstruction):
     ptrlo = operand_def(riscv.IntRegisterType)
     ptrhi = operand_def(riscv.IntRegisterType)
 
+    traits = frozenset([StaticInsnsRepresentation(".insn r 0x2b, 0, 0, x0, {0}, {1}")])
```

Comment:
What would be the best way to test this?

Reply:
I have tested this in verilator simulations, and once this is in we can convert the tests in ci to use this system over at `snax-mlir`. 
Without simulations, I'm not sure.

Reply:
Here a pytest test that checks that these operations have the exact string you would expect would be good, just to make sure that something in the formatting infrastructure doesn't break, for example